Snowflake.Data.Tests/IntegrationTests/SFDbDataReaderIT.cs

```diff
@@ -1617,19 +1617,185 @@ public void TestTimestampNtz(string testValue, int scale)
             using (var conn = CreateAndOpenConnection())
             {
                 DbCommand cmd = conn.CreateCommand();
-                
+
                 cmd.CommandText = $"select '{testValue}'::TIMESTAMP_NTZ({scale})";
                 using (SnowflakeDbDataReader reader = (SnowflakeDbDataReader)cmd.ExecuteReader())
                 {
                     ValidateResultFormat(reader);
 
                     reader.Read();
-                    
+
                     var expectedValue = DateTime.Parse(testValue);
 
                     Assert.AreEqual(expectedValue, reader.GetValue(0));
                 }
-                
+
+                CloseConnection(conn);
+            }
+        }
+
+        [Test]
+        public void TestGetEnumerator()
```

Comment:
You could also consider moving enumerator tests to a separate class because this one is already large.
Unless you have any reason stay it as is.

Reply:
I added it because it is an SFDbDataReader method, but I split it into another class. Thank you!!